src/support/mod.rs

```diff
@@ -213,7 +214,7 @@ pub fn enable_llvm_pretty_stack_trace() {
 /// A) Finds a terminating null byte in the Rust string and can reference it directly like a C string.
 ///
 /// B) Finds no null byte and allocates a new C string based on the input Rust string.
-pub(crate) fn to_c_str<'s>(mut s: &'s str) -> Cow<'s, CStr> {
+pub(crate) fn to_c_str(mut s: &str) -> Cow<'_, CStr> {
```

Comment:
Did this trigger a clippy warning? Would prefer to be explicit about the lifetimes here if not

Reply:
Yes, I believe it's https://rust-lang.github.io/rust-clippy/master/index.html#/needless_lifetimes